__tests__/endpoints.test.js

```diff
@@ -248,6 +248,22 @@ describe("endpoints", () => {
       expect(response.body.msg).toBe("Bad Request");
     });
   });
+
+  describe("GET /api/users", () => {
+    test("200: responds with an array of users", async () => {
+      const { body } = await request(app).get("/api/users").expect(200);
```

Comment:
Nicely done a small suggestion to take destructuring further - you can access the users by also destructuring like this:

const { body: {users} } = await request(app).get("/api/users").expect(200);